packages/components-js/src/pages/flyout-header-footer-scrolled-relative.html

```diff
@@ -0,0 +1,25 @@
+<style>
+  #app {
+    /* TODO: not needed as soon as all top layer components are refactored with iFrames */
+    transform: unset;
+  }
+</style>
+
+<p-flyout open="true" class="scroll content center">
+  <div slot="header">
+    <h1>Sticky Heading</h1>
+    <p>Sticky header text</p>
+  </div>
+  <div style="position: relative; z-index: 10; height: 600px; background-color: #61dafb"></div>
```

Comment:
should we move the inline styles to the style tag?

Reply:
I don't really see a lot of benefit in doing so.